streams/examples/main/java/org/apache/kafka/streams/examples/pageview/PageViewTypedJob.java

```diff
@@ -30,22 +31,35 @@
 import org.apache.kafka.streams.kstream.KStream;
 import org.apache.kafka.streams.kstream.KTable;
 import org.apache.kafka.streams.kstream.KeyValueMapper;
+import org.apache.kafka.streams.kstream.ValueJoiner;
 import org.apache.kafka.streams.kstream.Windowed;
 import org.apache.kafka.streams.StreamsConfig;
 
+import java.util.HashMap;
+import java.util.Map;
 import java.util.Properties;
 
+/**
+ * Implements the "PageView Stats" program that joins the pageview stream with
```

Comment:
What about:

> Demonstrates how to perform a join between a KStream and a KTable, i.e. an example of a stateful computation.
> In this example, we join a stream of pageviews (aka clickstreams) with a user profile table to compute the number of pageviews per user region.

I think, for all these examples, it is less important to summarize what exactly the example is doing ("we join pageview stream with a user table to achieve XYZ") but which techniques we're demonstrating ("how to join kstream with ktable").